content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/datasheet/datasheet.md

```diff
@@ -155,16 +155,17 @@ Now that you have gone through the basics of what you can do with the board, you
 
 
 ## Certifications
+
 ### Declaration of Conformity CE DoC (EU)
 We declare under our sole responsibility that the products above are in conformity with the
 essential requirements of the following EU Directives and therefore qualify for free movement
 within markets comprising the European Union (EU) and European Economic Area (EEA).
 Radio Equipment Directive (RED) 2014/53/EU
 
-We declare under our sole responsibility that the products above are in conformity with the
-essential requirements of the following EU Directives and therefore qualify for free movement
-within markets comprising the European Union (EU) and European Economic Area (EEA).
-Radio Equipment Directive (RED) 2014/53/EU
+### Declaration of Conformity to EU RoHS & REACH 211 01/19/2021
```

Comment:
remove this change

If I'm not wront you are going to duplicate a text

Reply:
The text is duplicated in the actual Datasheet. I am actually removing the duplicated text and adding the necessary information.

![image](https://user-images.githubusercontent.com/124149761/232011495-2545111b-9a6a-4a7a-bf7f-3522da6a6a78.png)

Reply:
I see! then proceed!